src/user/models/kyc/kyc.service.ts

```diff
@@ -175,13 +183,45 @@ export class KycService {
     const dataComplete = this.isDataComplete(user);
     if (!dataComplete) throw new BadRequestException('Ident data incomplete');
 
+    const [users, numberOfUsersWithSameInformation] = await this.userDataService.getUsersByInformation(user);
+    if (users) {
+      const completedUser = users.find((data) => KycCompleted(data.kycStatus));
+      const shouldSendLinkEmail = numberOfUsersWithSameInformation > 1;
+      if (shouldSendLinkEmail && completedUser) {
+        // this is not correct, where do we know. Which address is the correct one, if there are already linked addresses?
+        const existingAddress = completedUser.users[0].address;
+        // this should be fine, as a new user_data should have only one address
+        const newAddress = user.users[0].address;
```

Comment:
Comments can be removed?

Reply:
no basically they are still valid. I would propose to order based on create date and take the oldest one. currently I think it will be ordered by ID as this should be default, if I recall correctly.

Reply:
nvm - technically doesn't matter which address we are sending. But I felt it would be good to always have one main address, where we link to. and not sending emails with existing address A and for the same user on an additional address sending a different address.

Reply:
So what would you propose?

Reply:
I am now ordering for the oldest linked address, and we are always sending the oldest one

Reply:
Ok perfect 👍